pkgs/build-support/setup-hooks/patch-shebangs.sh

```diff
@@ -19,9 +19,8 @@ patchShebangs() {
     local newInterpreterLine
 
     find "$dir" -type f -perm -0100 | while read f; do
```

Comment:
This surprisingly worked well for us. However I wonder if this should be also just use https://stackoverflow.com/a/1120952

Reply:
This can also go in different pull request also I thought it might be easy enough to change here as well.